src/NewProject/source_grid.cpp

```diff
@@ -14,12 +14,12 @@ using namespace FOEDAG;
 
 sourceGrid::sourceGrid(QWidget *parent) : QWidget(parent) {
   m_lisFileData.clear();
-  m_btnAddFile = new QPushButton(tr("AddFile"), this);
+  m_btnAddFile = new QPushButton(tr("Add File"), this);
   connect(m_btnAddFile, &QPushButton::clicked, this, &sourceGrid::AddFiles);
-  m_btnAddDri = new QPushButton(tr("AddDir"), this);
+  m_btnAddDri = new QPushButton(tr("Add Dir"), this);
   connect(m_btnAddDri, &QPushButton::clicked, this,
           &sourceGrid::AddDirectories);
-  m_btnCreateFile = new QPushButton(tr("CreateFile"), this);
+  m_btnCreateFile = new QPushButton(tr("Create File"), this);
```

Comment:
To stay consistent with these changes, "MoveUp" and "MoveDown" on lines 29 and 32 should probably be updated to "Move Up" and "Move Down".

Reply:
Any reason to use shortform "Dir" ?

Reply:
My guess is the original developer was just trying to save space as a toolbar with "Create Directory" is going to have a large portion taken up by that one button.

Reply:
Thanks. @zaryabkhan-rs please add a screenshot here on how it looks visually with "Add Directory".

Reply:
Here it is, 
![dir](https://user-images.githubusercontent.com/109368108/180375012-f9a20c34-926a-4320-9c37-ed70aaf6d209.png)

Reply:
Thanks. Please change it to "Add Directory".

Reply:
Okay got it.

Reply:
> To stay consistent with these changes, "MoveUp" and "MoveDown" on lines 29 and 32 should probably be updated to "Move Up" and "Move Down".

@skyler-rs Yeah, Now I have changed it.